src/View/Startup.fxml

```diff
@@ -1,48 +1,58 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import javafx.scene.control.Button?>
-<?import javafx.scene.image.ImageView?>
-<?import javafx.scene.layout.ColumnConstraints?>
-<?import javafx.scene.layout.GridPane?>
-<?import javafx.scene.layout.RowConstraints?>
-<?import javafx.scene.text.Font?>
+<?import javafx.scene.shape.*?>
```

Comment:
Wildcard imports are not consistent with our project coding style.  Please add only the imports that are actually being used, and please make sure that the import list properly sorted in alphabetical order.